tests/test_playwright.py

```diff
@@ -143,3 +122,53 @@ def check_colors():
                 expect(el).to_have_css("color", hover_color)
 
     _check_test_site(site_name, site_path, check_colors)
+
+
+# ------------------------- Test functions: layout & components -----------------------
+
+
+def test_breadcrumb_expansion(
```

Comment:
the GitHub diff view does a bad job of aligning this with the old code; can you confirm that the breadcrumb-related tests are unchanged? In diff view they show up as fully deleted above, then fully added here (making it hard to assess if anything changed)